05-ignore.md

```diff
@@ -6,13 +6,25 @@ minutes: 5
 ---
 > ## Learning Objectives {.objectives}
 > 
-> *   Configure Git to ignore specific files,
->     and explain why it is sometimes useful to do so.
+> *   Explain why it is sometimes useful to have Git ignore specific files
+> *   Configure Git to do so
+>      
 
-What if we have files that we do not want Git to track for us,
-like backup files created by our editor
-or intermediate files created during data analysis.
-Let's create a few dummy files:
+### Why it is sometimes useful to *not* track files
+
+Git is incredible useful tool for tracking the changes to your code. However, it's not necessary to the track the changes of *every* file associated with a project. Sometime there are non-code files in your project directory for which you might not want to track changes.  For example:
```

Comment:
I think the second and third sentences say mostly the same thing. What about something shorter like: 

"However, sometimes it's not necessary to the track the changes of *every* file in your working directory. For example, you might want to ignore changes in certain types of files, such as:"

I used working directory instead of project directory because is the terminology we use in the rest of the lesson. Also, I dropped "associated with the project" because we want to emphasize that the repo is restricted to a directory and "associated with" maybe another file outside this directory.